node-scripts/src/pipeline3/host-preparation.js

```diff
@@ -17,8 +17,6 @@ const config = require(cst.CONFIGPATH);
 const db = require(cst.DBPATH);
 
 const scripts = require("./scripts");
-
-const secrets = config.getSecrets();
```

Comment:
Please keep this line. Even though it is not used, I like that it is documented how it can be leveraged on.
you can comment it though:
```
// Fetch secrets:
// const secrets = config.getSecrets();
```